src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Doctrine\Common\Annotations\CachedReader;
+use Doctrine\Common\Annotations\Reader;
+use Doctrine\Common\Cache\ArrayCache;
+use Doctrine\Common\Cache\FilesystemCache;
+use Symfony\Bundle\FrameworkBundle\CacheWarmer\AnnotationsCacheWarmer;
+use Symfony\Component\Cache\Adapter\PhpArrayAdapter;
+use Symfony\Component\Cache\DoctrineProvider;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('annotations.reader', AnnotationReader::class)
+            ->call('addGlobalIgnoredName', [
+                'required',
+                service('annotations.dummy_registry'), // dummy arg to register class_exists as annotation loader only when required
+            ])
+
+        ->set('annotations.dummy_registry', AnnotationRegistry::class)
+            ->call('registerUniqueLoader', ['class_exists'])
+
+        ->set('annotations.cached_reader', CachedReader::class)
+            ->args([
+                service('annotations.reader'),
+                inline_service(ArrayCache::class),
+                abstract_arg('Debug-Flag'),
+            ])
+
+        ->set('annotations.filesystem_cache', FilesystemCache::class)
+            ->args([
+                abstract_arg('Cache-Directory'),
+            ])
+
+        ->set('annotations.cache_warmer', AnnotationsCacheWarmer::class)
+            ->args([
+                service('annotations.reader'),
+                param('kernel.cache_dir').'/annotations.php',
+                '#^Symfony\\(?:Component\\HttpKernel\\|Bundle\\FrameworkBundle\\Controller\\(?!.*Controller$))#',
```

Comment:
The slashes must occur 4 times as they are escape chars in php and need to be doubled to get reduced to 2 as regex.